src/controllers/trust.involved.controller.ts

```diff
@@ -34,6 +36,7 @@ type TrustInvolvedPageProperties = {
     beneficialOwnerTypeTitle: Record<string, string>;
     trusteeTypeTitle: Record<string, string>;
     trusteeType: typeof TrusteeType;
+    trusteeData: GeneralTrustee[];
```

Comment:
Should `trusteeData` maybe be called `individualTrusteeData` or something similar because the other 2 trustee types (legal entity(corporate) and historical(former) don't fit into `GeneralTrustee `

Reply:
Yes it should forgot that change thanks